src/Manager.js

```diff
@@ -1,56 +1,8 @@
+import Hotel from '../src/Hotel';
 
-class Manager {
+class Manager extends Hotel {
   constructor(userData, roomData, bookingsData) {
-    this.users = userData;
```

Comment:
Looks like a pretty big refactor, and glad to see you are working through where you want functional repsonility to lie betwen a parent and child class.